telnyx_common/src/main/java/com/telnyx/webrtc/common/TelnyxViewModel.kt

```diff
@@ -293,6 +293,7 @@ class TelnyxViewModel : ViewModel() {
 
     /**
      * Updates the debug mode for the current profile.
+     * When this option is on it allows the SDK to collect WebRTC debug information.
```

Comment:
We should mention that for this, it is stored on the portal. As opposed to collecting it and showing it via a callback (like we do for invite)